core/src/components/menu/menu.scss

```diff
@@ -59,15 +59,72 @@
 }
 
 :host(.menu-side-start) .menu-inner {
+  /**
+    * Menu does not cover the whole screen so we need to set the safe area for the
+    * side that touches the screen edge only. Since safe area is not logical, it
+    * needs to be applied to the correct side depending on the language direction.
+    * Otherwise, the content will have less space on both sides.
+    *
+    * LTR:
+    * The left side of the menu touches the screen edge. Padding is applied to the
+    * left side of the menu. No changes are needed since the value has been set.
```

Comment:
This phrasing feels strange to me, and I'm not totally clear on "no changes are needed since the value has been set" means. I think you were going for something like this?
> The left side of the menu touches the screen edge. The safe area has already been set elsewhere [specify where?], so extra padding does not need to be applied here.

Ditto for the corresponding comment on `side-end`.